Sources/TuistKit/Services/RunService.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import RxSwift
+import TSCBasic
+import TuistAutomation
+import TuistCore
+import TuistSupport
+
+enum RunServiceError: FatalError {
+    case appNotFound(targetName: String)
+    case simulatorNotFound
+    case targetNotFound(forSchemeName: String)
+
+    var description: String {
+        switch self {
+        case let .appNotFound(targetName):
+            return "Couldn't find any apps built for \(targetName)"
+        case .simulatorNotFound:
+            return "Could't find simulator"
```

Comment:
An error like this would leave the user without knowing what to do. What about giving the user some tips on what to do in this case? What could be the reason for a simulator not being found?